go-controller/pkg/config/const.go

```diff
@@ -21,6 +21,7 @@ const (
 	// priority of logical router policies on the OVNClusterRouter
 	EgressFirewallStartPriority           = "10000"
 	MinimumReservedEgressFirewallPriority = "2000"
+	HybridOverlayNodeReroutePriority      = "1006"
```

Comment:
Just to understand: this patch is creating logical router policies for the pod subnet on every node -> hybrid overlay cluster subnet. Do we really care about specifying any particular value at all for the logical router policy? It's only going to match a very particular type of network traffic so it's unlikely (impossible?) to match several logical router policies, which thus mean: couldn't we set it to `1` and be fine as well? I am not specifically looking to change the value from `1006`, but more so understand what is being done....even though if that is the case I would change it to `1` and add a doc-bloc mentioning this so that it's clearer than having multiple priorities which indicate some form of routing decision.